backend/src/main.rs

```diff
@@ -32,6 +33,11 @@ async fn main() -> std::io::Result<()> {
         .await
         .expect("Error building a connection pool");
 
+    sqlx::migrate!("./migrations")
+        .run(&pool)
+        .await
+        .expect("Error running migrations");
```

Comment:
should I make this config driven? ([example](https://github.com/JoeyMckenzie/realworld-rust-axum-sqlx/blob/main/crates/conduit-infrastructure/src/connection_pool.rs#L10))

From a local dev perspective this is convenient, but for production deploys I think having more explicit controls around when migrations are run is better.

Reply:
Sure thing - resolved by [d16869a](https://github.com/Arcadia-Solutions/arcadia/pull/85/commits/d16869a50e569dcdbfbf165bf033d7e134273815)